src/particles/text/component.tsx

```diff
@@ -64,26 +64,45 @@ export const getTextTag = (variant?: string): TextTag => {
 interface IStyledTextProps {
   theme: ITextTheme;
   alignment?: TextAlignment;
+  clipToLines?: number;
 }
 
 const StyledText = styled.span<IStyledTextProps>`
   ${(props: IStyledTextProps): string => themeToCss(props.theme)};
   ${(props: IStyledTextProps): string => (props.alignment ? `text-align: ${props.alignment}` : '')};
+
+  &.singleLine {
```

Comment:
is this just one line or multiple?

Reply:
i think it would be better to create two classes `singleLine` and `fixedLines` and set the class based on the prop in Text

Reply:
ooh I forgot to rename the class, but it works for multiple lines as well

Reply:
done